<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BufferClass</class>
 <widget class="QMainWindow" name="BufferClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditFilename">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRadius">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>缓冲半径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>输出图层：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>···</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilename">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输出文件名：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>···</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>输入图层：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>缓冲区分析</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRadius">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
